pkgs/development/tools/misc/dwz/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchurl, elfutils }:
+
+stdenv.mkDerivation rec {
+  pname = "dwz";
+  version = "0.14";
+
+  src = fetchurl {
+    url = "https://www.sourceware.org/ftp/${pname}/releases/${pname}-${version}.tar.gz";
+    sha256 = "07qdvzfk4mvbqj5z3aff7vc195dxqn1mi27w2dzs1w2zhymnw01k";
+  };
+
+  nativeBuildInputs = [ elfutils ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p "$out/bin"
+    cp dwz "$out/bin"
+
+    runHook postInstall
+  '';
```

Comment:
```suggestion
  makeFlags = [ "prefix=${placeholder "out"}" ];
```

Reply:
Hmm, this should solve the [man page issue](https://github.com/NixOS/nixpkgs/pull/136078#pullrequestreview-741149817).

Reply:
Thanks! This is much cleaner and installs both the executable and man page.